_data/simple-icons.json

```diff
@@ -10742,6 +10742,11 @@
             "source": "https://www.okta.com/press-room/media-assets/",
             "guidelines": "https://www.okta.com/terms-of-use-for-okta-content/"
         },
+        {
+            "title": "OKX",
+            "hex": "121212",
+            "source": "https://www.okx.com"
```

Comment:
I'm not seeing any SVGs on this website - did you get it from https://commons.wikimedia.org/wiki/File:OKX_logo.svg?

Reply:
Favicon and [application icon](https://www.okx.com/fr/download) are also using rounded corners:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/41993847-ca07-4231-9300-f49201622ce5)

Reply:
@uncenter Yes, I took the logo from the wiki and cut it in Inkscape
@service-paradis you are right, round corners added in ratio from application icon